task5/src/main/java/com/clouway/core/UserRepository.java

```diff
@@ -12,4 +12,10 @@
   boolean authenticate(String email, String password);
 
   void deleteAll();
+
+  void makeDeposit(double amount, String email);
```

Comment:
`deposit` is enough. In this case there is nothing wrong. But it is always better if naming can be shorter. This is suggestion.